packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -15,6 +16,9 @@ const chainIds = {
   [Network.Testnet]: 'deprecated',
 }
 
+register9Rheader(axios)
+register9Rheader(cosmosclient.config.globalAxios)
+
```

Comment:
Mock URL https://stagenet-thornode.ninerealms.com/thorchain/tx/99C97CB3DAC2BABBF5EF2938C15E8D3AEA55A815BBD04FCB53D97FA044E941CC  instead of add headers in tests

Reply:
resolved